requirements.adoc

```diff
@@ -53,17 +67,46 @@ specific patient.
 CONTEXT::
 The radiology order is sent to the radiology department for execution.
 
+
+Judy::
+
+```
+There is a use case where an outside image(done from a different facility is obtained and brought by the patient
+and needs to be read at the local facility).Therefore there is an order for "Outside Interpretation" which would include
+1. Patient Name (First , Middle and Last Name)/ MRN
+2. Study type eg CT head
+3. Name of facility
+4. Whether report is available..
+
+This would create a new study that is complete (after images are uploaded to the PACS system) and a user can claim a report. The label for this study would be
+OUTSIDE CT HEAD .. etc
+```
+
 COMPONENTS::
 Its main components are
 
-* patient
-* urgency
-* start date
+* patient details (MRN/Name)
+* urgency (These are priorities. Some people use numbers eg 0 for must be read immediately, 2 for ED studies , 4 for inpatients and 5 for outpatient /non emergent studies). These could be set up as a global property that can be changed by the person creating
+the order , and this field created as a drop down.
+* start date -- I would change this to Study Date...
 * orderable (what kind of test should be done? i.e. "CT of the
-Abdomen" or "R/O Pulmonary Embolism")
+Abdomen" I deleted the R/O Pulmonary Embolism is an order reason
```

Comment:
**Orderable**
The orderable is what you can currently see labeled as "Imaging Procedure" in the order entry page. This one is already configured via global properties, where you enter a list of concept classes. The orderer can then only select a concept which is part of these concept classes.

Should I rename the label "Imaging Procedure" to "Orderable"?

_Clarification_
What I wanted to make clear is that the orderable is not necessarily the exact radiological procedure that will be done at the modality. The orderable is coming from the referring phyisician or ordering clinician as you named them. Therefore I added "R/O Pulmonary Embolism" to make it clear to others that this can be anything that the radiology department says it can test (and then configures as orderable via global properties as described above). The radiology department then decides how this orderable is done in terms of radiological procedures. Therefore the separation also in the IHE model of orders -> requested procedure -> ....

Do you agree with me here?

**Order Reason**
I wanted to ask you about the order reason:
the openmrs-core Order has orderReason from Concepts and orderReasonNonCoded as String
(see https://github.com/openmrs/openmrs-core/blob/2.0.x/api/src/main/java/org/openmrs/Order.java#L80)

I want to add those fields to our Order entry page so the orderer can enter those. Good?